metricbeat/module/apache/status/_meta/fields.yml

```diff
@@ -4,6 +4,7 @@
   description: >
     `status` contains the metrics that were scraped from the Apache status page.
   fields:
+    # Apache hostname
```

Comment:
Hmmmm, I see how this can become a problem.

My take on it is that yes, this module should overwrite `host.hostname`, even if it's already set.

What we're wrestling with here is actually what `observer` is for. Metricbeat is acting as an observer, so whatever host details of Metricbeat belong under `observer`. The events it gathers are about potentially external hosts, and belong in `host`.

Now how/when to determine that is where it gets tricky... Should it be a user-facing option to identify the Metricbeat instance as an observer? I don't think people will understand what this is about.

Or we could detect this, and do the right thing automatically. But that may be confusing when the Metricbeat host's data is sometimes under `host` and sometimes under `observer`.

Perhaps after all we should simply always make Metricbeat an observer?

Not quite sure, I'll think about this some more...

Reply:
To not get blocked on the above discussion, what do you think about putting this into `service.hostname`?

Reply:
Skipping for now.